config/locales/simple_form.en.yml

```diff
@@ -90,10 +90,10 @@ en:
         text: Custom warning
         type: Action
         types:
-          disable: Disable login
-          none: Do nothing
-          silence: Silence
-          suspend: Suspend and irreversibly delete account data
+          disable: Freeze
+          none: Send a warning
+          silence: Limit
```

Comment:
I do since "Suspend and irreversibly delete account data" being changed changes the balance of things.

Reply:
I am uncomfortable with changing “Silence” into “Limit” here, I think it's a bit less clear what it does, and it might be out of sync with other parts of the codebase and documentation.

Reply:
End-users have been received e-mails titled "Your account has been limited" ever since those e-mails became a thing. I think it would be good to converge the terminology between end-user/admin views especially since the shadowban aspect of "silence" has been reworked to be obvious to the end-user.

Reply:
Hmm alright.